app/models/categoryModels.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../../db/connection");
+
+const getCategoriesModel = () => {
+  return db.query("SELECT slug, description FROM categories").then((result) => {
+    if (result.rows) {
+      return result.rows;
+    } else {
+      return Promise.reject({ code: 204, msg: "there is no content" });
+    }
```

Comment:
I don't think this Promise would ever reject, as you'd either get categories back or an empty array, which would still be a truthy value. So this isn't really needed